docker-storage-setup.sh

```diff
@@ -839,6 +853,123 @@ get_existing_storage_driver() {
   return 1
 }
 
+docker_root_volume_exists() {
+    lvs $VG/$DOCKER_ROOT_LV_NAME > /dev/null 2>&1 && return 0
+    return 1
+}
+
+# Check if docker root volume is already mounted.
+# This return 0 if volume is already mounted on $DOCKER_ROOT_DIR. Returns 1
+# if volumes needs mounting. On all other error conditions it exits.
+
+docker_root_lv_mounted() {
+  local mounts target
```

Comment:
I think we can modify this function to return location of mount point of docker volume. It will return 1 in case of error. Return 0 in case of success or if volume is not mounted. If volume is not mounted, will return 0 and echo empty string.